### PR TITLE
chore: add output for ID of VPC internet gateway

### PR DESCRIPTION
**Pull request by JoseFMP**
### Description

Adds an output for the ID of the internet gateway that is used in the VPC. This will be consumed by other modules.

---
This change is [Reviewable](https://reviewable.io/reviews/honestbank/terraform-aws-vpc/6)

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -22,3 +22,8 @@ output "vpc_id" {
   description = "AWS identifier for the VPC provisioned by this module"
   value       = module.aws-vpc.vpc_id
 }
+
+output "igw_id" {
```
Comment (jai) [inline outputs.tf:26]: I think it would be helpful to add `igw_arn` as well just in case the name isn't unique?

**Review by jai — CHANGES_REQUESTED** [review]

File: outputs.tf (hunk 1 above)
Comment (JoseFMP) [reply outputs.tf:26]: We just added it

**Review by poom — APPROVED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Release
+        uses: cycjimmy/semantic-release-action@v2
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          extra_plugins: |
+            @semantic-release/git@7.0.18
```
Comment (jai) [inline .github/workflows/release.yml:17]: FYI these versions are pretty old I think, just saw a recent PR from @maetolay updating these in the .github repo

**Review by jai — APPROVED** [review]
LGTM

File: .github/workflows/release.yml (hunk 2 above)
Comment (JoseFMP) [reply .github/workflows/release.yml:17]: We actually copied it from another repo were @bludot had fixed it. Wondering if we could somewhat pin this version across all repos.

**Pull request merged**